backend/src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
-server.port=${PORT:8080}
\ No newline at end of file
+server.port=${PORT:8080}
+
+app.jwt.secret=${JWT_SECRET:test-secret}
+
+spring.data.mongodb.database=Cocktail
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
```

Comment:
Instead of these three properties I would use `spring.data.mongodb.uri`. As a value you can use `${MONGODB_URI:mongodb://localhost:27017/Cocktail}`. Than everything is prepared for Heroku.